### PR TITLE
Added French

### PR DESCRIPTION
**Pull request by lipis**
Soon.. https://github.com/wireapp/wire-webapp/pull/207

We need more translators: https://crowdin.com/project/wire-cs/fr# :)
(no need to translating the website)

### CODE REVIEW

File: electron/locale/locale.js [hunk 1]
```diff
@@ -25,6 +25,7 @@ const config = require('./../js/config');
 const init = require('./../js/lib/init');
 const de = require('./strings-de');
 const en = require('./strings-en');
+const en = require('./strings-ru');
```
Comment (ConorIA) [inline electron/locale/locale.js]: shouldn't this be strings-fr?

**Comment by ConorIA** [comment]
Not related to this PR, but Spanish is now fully translated, so you can add that if that is on the teams radar. :)

File: electron/locale/locale.js (hunk 1 above)
Comment (lipis) [inline electron/locale/locale.js]: yep!

**Comment by lipis** [comment]
New PR coming soon

**Pull request closed without merging**